cmd/redplex/redplex.go

```diff
@@ -43,11 +44,14 @@ func main() {
 
 	logrus.SetLevel(level)
 
+	password := os.Getenv("REDIS_PASSWORD")
+	sentinelPassword := os.Getenv("SENTINEL_PASSWORD")
```

Comment:
For consistency, can you allow these to be provided as flags and use `.OverrideDefaultFromEnvar("REDIS_PASSWORD")` to optionally pull them from the environment variable?

Reply:
@connor4312 I had them as flags initially, but that encourages bad behaviour as commandline args can be read by any user on a system and therefor supplying credentials as arguments is considered bad practice

Reply:
Yea, that's fair, the main reason I want it there is for documentation with --help. If you can manually add it to the help output that would also work :) Something like this maybe

```go
kingpin.UsageTemplate(kingping.DefaultUsageTemplate + "\r\nMore help text here")
```